modules/core/shared/src/main/scala/gem/math/ProperMotion.scala

```diff
@@ -170,9 +170,21 @@ object ProperMotion {
   }
   // scalastyle:on method.length
 
-  implicit val EqProperMotion: Eq[ProperMotion] =
-    Eq.fromUniversalEquals
+  implicit val OrderProperMotion: Order[ProperMotion] = {
 
+    implicit val MonoidOrder: Monoid[Order[ProperMotion]] =
+      Order.whenEqualMonoid[ProperMotion]
+
+    def order[A: Order](f: ProperMotion => A): Order[ProperMotion] =
+      Order.by(f)
+
+    order(_.baseCoordinates)  |+|
+      order(_.epoch)          |+|
+      order(_.properVelocity) |+|
+      order(_.radialVelocity) |+|
+      order(_.parallax)(catsKernelStdOrderForOption(Angle.SignedAngleOrder))
```

Comment:
@tpolecat This is the pattern I was looking for yesterday on the chatter.   It seems okay to me for a case class with many fields.

Reply:
Why's parallax different?

Reply:
Parallax is just an `Angle` and I didn't want to define an implicit `Order` for `Angle` in its companion, though I guess I could define one in this local scope and make the implementation a bit nicer.

The reason to not define a standard `Order[Angle]` was that it isn't always clear whether you want that ordered from [180, 180) or [0, 360).  Or that was my thinking at any rate.

Reply:
Can you add a comment explaining why you're not using `Order.by` here? To avoid boxing I assume?

Reply:
I put the reason in a response to Carlos about how `Order` is defined for tuples, but I can add it to the code if you think it is valuable.  Maybe it's not a good reason though?  I'm happy to switch to `Order.by`.

Reply:
I think it might be useful to have in the code since we're not doing it this way elsewhere.